crates/wash-lib/src/lib.rs

```diff
@@ -28,14 +28,21 @@ pub mod parser;
 #[cfg(feature = "start")]
 pub mod start;
 
+#[cfg(feature = "nats")]
 pub mod actor;
+#[cfg(feature = "nats")]
 pub mod capture;
 pub mod common;
+#[cfg(feature = "nats")]
 pub mod config;
+#[cfg(feature = "nats")]
```

Comment:
Does context need to be gated behind the `nats` feature flag for this to work?

Reply:
yes, because it depends on `config` which is behind the `nats` flag.